src/picosvg/picosvg.py

```diff
@@ -29,6 +29,11 @@
 FLAGS = flags.FLAGS
 
 
+flags.DEFINE_bool(
+    "drop_unsupported",
```

Comment:
since we already have `allow_text`, I would call this `allow_unsupported` -- the latter is basically a superset of the former

Reply:
ehm sorry got confused, nevermind, this one drops whereas the allow_text one keeps 😅